### PR TITLE
docs(en-US): manage users -> You dont need a user table in the database

### PR DESCRIPTION
**Pull request by IceHe**
- Add the page "Manage users"
    - Add the section "You don't need a user table in the database"

### CODE REVIEW

File: docs/docs/recipes/manage-users/README.md [hunk 1]
```diff
@@ -3,3 +3,63 @@ sidebar_position: 6
...
 
 # 🧑‍🚀 Manage users
+
+## You don't need a user table in the database
+
+Before taking Logto as your identity solution, you may think about how to organize users and related info.
+Traditionally, developers create a user table in a database containing the user info, then create a many-to-one relation in other tables, with a `user_id` column pointing to the user table.
+
+Taking Google as an example for other identity providers, you may create a column named `google_id` to associate the users who "Sign in with Google".
+For Logto, you may also intend to create a column `logto_id`, but it is unnecessary.
```
Comment (gao-sun) [inline docs/docs/recipes/manage-users/README.md:13]: it doesn't like a proper example.

- i didn't see the connection between google and logto
- didn't understand what's logto_id for

File: docs/docs/recipes/manage-users/README.md [hunk 2]
```diff
@@ -3,3 +3,63 @@ sidebar_position: 6
...
+Logto is more than an identity provider; it is a solution that can take charge of all user info.
+
+**Logto best practice**: Forget the user table, and store your user info in Logto.
+
+1. Create a `user_id` column in other tables, and save Logto's `user.id`.
+   See [Identify a user](#identify-a-user) for details.
+2. Call Logto's Management API for CRUD: create, read, update, delete user info, and so on.
+   See [Users API](https://docs.logto.io/api/#tag/Users) for details.
+3. Save any additional user info to custom data.
+   See [Update a user](#update-a-user) for details.
```
Comment (gao-sun) [inline docs/docs/recipes/manage-users/README.md:23]: 1 looks not related to 2&3

**Review by gao-sun — CHANGES_REQUESTED** [review]
will be better to have 2 specific and different scenarios to demonstrate how to achieve this. also the structure is not expected. please see other recipes for reference

File: docs/docs/recipes/manage-users/README.md (hunk 2 above)
Comment (IceHe) [reply docs/docs/recipes/manage-users/README.md]: converted the ordered list to the bullet list

**Review by IceHe — COMMENTED** [review]
Will create another PR to deal with the structure of Manage Users readme
